system/lib/libcxxabi/src/cxa_exception_emscripten.cpp

```diff
@@ -14,11 +14,12 @@
 #include <stdint.h>
 #include <stdlib.h>
 
-#if defined(__USING_EMSCRIPTEN_EXCEPTIONS__) ||                                \
-  defined(__USING_WASM_EXCEPTIONS__)
-
-// TODO: Remove this when Rust doesn't need it anymore.
+#if !defined(__USING_WASM_EXCEPTIONS__)
+// Until recently, Rust's `rust_eh_personality` for emscripten referred to this
+// symbol. If Emscripten doesn't provide it, there will be errors when linking
+// rust. The rust personality function is never called so we can just abort.
```

Comment:
Sorry for nits, but can we mention that we need this for old versions of Rust? By just looking at this comment I would be wondering like, "If it has been recently removed from Rust why do we need this?"